version.json

```diff
@@ -1,88 +1,88 @@
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-dependencies-js": {
             "package": "extensions/applicationinsights-dependencies-js/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-perfmarkmeasure-js": {
             "package": "extensions/applicationinsights-perfmarkmeasure-js/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-properties-js": {
             "package": "extensions/applicationinsights-properties-js/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-common": {
             "package": "shared/AppInsightsCommon/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-core-js": {
             "package": "shared/AppInsightsCore/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-chrome-debug-extension": {
             "package": "tools/chrome-debug-extension/package.json",
-            "release": "0.4.6"
+            "release": "0.4.7"
         },
         "applicationinsights-web-config": {
             "package": "tools/config/package.json",
-            "release": "0.0.10"
+            "release": "0.0.11"
         },
         "@microsoft/applicationinsights-example-aisku": {
             "package": "examples/AISKU/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-example-dependencies": {
             "package": "examples/dependency/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-example-shared-worker": {
             "package": "examples/shared-worker/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-teechannel-js": {
             "package": "channels/tee-channel-js/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-test-module-type-check": {
             "package": "AISKU/Tests/es6-module-type-check/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-cfgsync-js": {
             "package": "extensions/applicationinsights-cfgsync-js/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/applicationinsights-example-cfgsync": {
             "package": "examples/cfgSync/package.json",
-            "release": "3.0.6"
+            "release": "3.0.7"
         },
         "@microsoft/1ds-post-js": {
```

Comment:
Note: explicitly not updating the 1ds packages as we won't be publishing them and this bug is specific to AI only

Reply:
It's also not included in the "list" of changes as its package.json only references 1ds-core-js